### DISCUSSION TITLE
GX Error #48 DVCC with incompatible firmware: Warning

### DISCUSSION THREAD

**Discussion by dd6uw** [Louisvdw/dbus-serialbattery#795] [Q&A]
Hi,
mostly every in my System works fine,
but my Cerbo GX always greets me with wollowing message:

`GX Error #48 DVCC with incompatible firmware: Warning`

I have
Daly Serial Battery Monitor Firmware Version  	1.0.20230531
Victron Cerbo GX v 3.01
any Ideas?

**Comment by mr-manuel** [comment]
Did you check [DVCC - Distributed Voltage and Current Control](https://www.victronenergy.com/media/pg/Cerbo_GX/en/dvcc---distributed-voltage-and-current-control.html)?

**Reply by dd6uw** [reply]
sorry, was my fault, i worked with an old Firmware on my mppt. now it is ok.
